### PR TITLE
Allow configuring locale in the Faker connector

### PR DESCRIPTION
**Pull request by nineinchnick**
## Description

Allow configuring locale in the Faker connector.

## Release notes

```markdown
## Faker
* Allow configuring locale via the `aker.locale` config property. ({issue}`24152`)
```

### CODE REVIEW

File: plugin/trino-faker/src/test/java/io/trino/plugin/faker/FakerQueryRunner.java [hunk 1]
```diff
@@ -47,6 +50,12 @@ protected Builder()
                     .build());
         }
 
+        public Builder setProperties(Map<String, String> properties)
```
Comment (ebyhr) [inline plugin/trino-faker/src/test/java/io/trino/plugin/faker/FakerQueryRunner.java:53]: Rename this method to `setConnectorProperties`. The builder already has `setExtraProperties` and  `addCoordinatorProperty`. The current method name looks confusing.

Comment (raunaqmorarka) [reply plugin/trino-faker/src/test/java/io/trino/plugin/faker/FakerQueryRunner.java:53]: setProperties -> setFakerProperties

Comment (nineinchnick) [reply plugin/trino-faker/src/test/java/io/trino/plugin/faker/FakerQueryRunner.java]: Thank you, these are great suggestions!

**Review by raunaqmorarka — APPROVED** [review]

File: docs/src/main/sphinx/connector/faker.md [hunk 2]
```diff
@@ -46,6 +47,9 @@ The following table details all general configuration properties:
     that allows them. Defaults to `0.5`.
 * - `faker.default-limit`
   - Default number of rows in a table. Defaults to `1000`.
+* - `faker.locale`
+  - Default locale for generating character based data, specified as a IETF BCP
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/faker.md:51]: character-based

File: docs/src/main/sphinx/connector/faker.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ For example, to generate data in the `generator` catalog, create the file
 connector.name=faker
 faker.null-probability=0.1
 faker.default-limit=1000
+faker.locale=pl-PL
```
[the remark this answers is not in the document]
Comment (nineinchnick) [reply docs/src/main/sphinx/connector/faker.md:24]: https://github.com/trinodb/trino/pull/24169

**Pull request merged**